bls/src/main/swig/bls-signatures.i

```diff
@@ -0,0 +1,243 @@
+%module DASHJBLS
+%{
+#include "bls-signatures/src/bls.hpp"
+#include "bls-signatures/src/chaincode.hpp"
+#include "bls-signatures/src/elements.hpp"
+#include "bls-signatures/src/extendedprivatekey.hpp"
+#include "bls-signatures/src/extendedpublickey.hpp"
+#include "bls-signatures/src/hdkeys.hpp"
+#include "bls-signatures/src/hkdf.hpp"
+#include "bls-signatures/src/privatekey.hpp"
+#include "bls-signatures/src/schemes.hpp"
+#include "bls-signatures/src/threshold.hpp"
+#include "bls-signatures/src/util.hpp"
+#include "threshold-v1.hpp"
+using namespace bls;
+%}
+
+%include "std_vector.i"
+%include "std_string.i"
+%include "bytes.i"
+typedef unsigned char uint8_t;
+typedef unsigned int uint32_t;
+typedef long size_t;
+namespace std {
+  %template(Uint8Vector) vector<uint8_t>;
+  %template(Uint8VectorVector) vector<vector<uint8_t>>;
+  %template(PrivateKeyVector) vector<bls::PrivateKey>;
+  %template(G1ElementVector) vector<bls::G1Element>;
+  %template(G2ElementVector) vector<bls::G2Element>;
+}
+
+namespace bls {
+  %copyctor G1Element;
+  %copyctor G2Element;
+  %ignore BLS::SetSecureAllocator;
+  %ignore G1Element::FromNative;
+  %ignore G1Element::ToNative;
+  %ignore G2Element::FromNative;
+  %ignore G2Element::ToNative;
+  %ignore GTElement::FromNative;
+  %ignore GTElement::ToNative;
+  %ignore PrivateKey::PrivateKey(PrivateKey &&k);
+  %ignore Util;
+
+  %rename("%(lowercamelcase)s", %$isfunction) "";
+  %rename ("$ignore", fullname=1) CoreMPL::Aggregate(const vector<Bytes>& signatures);
+  %rename ("$ignore", fullname=1) CoreMPL::AggregateVerify(const vector<Bytes>& pubkeys, const vector<Bytes>& messages, const Bytes& signature) override;
+  %rename ("$ignore", fullname=1) CoreMPL::AggregateVerify(const vector<G1Element>& pubkeys, const vector<Bytes>& messages, const G2Element& signature) override;
```

Comment:
Here, all methods that have `vector<Bytes>` are ignored.  This vector template could not be wrapped in java.